cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -163,7 +164,11 @@ def test_vmware_reconfigure_vm_controller_type():
             3.Reconfigure VM opion should be enabled
             4.Controller type should be listed
     """
-    pass
+    vms_collections = appliance.collections.infra_vms
+    vm = vms_collections.instantiate(name='cu-24x7', provider=provider)
```

Comment:
**Required** I think we should check to make sure this VMs exists before continuing, there is a chance that this may not exist on the provider. In addition, we should have `cap_and_util` as a required field for the provider.

Reply:
Will add this.

Reply:
still needs to be added here

Reply:
@kedark3 was thinking more like:
```if not vm.exists_on_provider:
        pytest.skip("Skipping test, cu-24x7 VM does not exist")```

I wouldn't say we want a fail test here, because the test isn't failing if the vm is not on the provider.